src/components/pages/Login.js

```diff
@@ -1,10 +1,102 @@
-import { Flex } from "antd"
-import { Logo } from "../atoms/Logo"
+import { Button, Col, Flex, Form, Grid, Input, Row } from "antd";
+import colors from "../../utils/colors";
+import { LockOutlined, UserOutlined } from "@ant-design/icons";
+import { LogoH } from "../atoms/LogoH.";
+import useLogin from "../../hooks/useLogin";
+const { useBreakpoint } = Grid;
+
+const loginCardStyle = { backgroundColor: '#FFF', borderRadius: '8px', minHeight: '450px'  }
+const loginFormTitleStyle = { textAlign: 'center', marginBottom: '2rem', fontWeight: 'bold', color: colors.secondary}
+const flexCentered = { display: 'flex', flexDirection: 'column', justifyContent: 'center', alignItems: 'center' }
 
 export const Login = () => {
+  const { loginError, onFinish } = useLogin()
+
+  const { sm, md, lg } = useBreakpoint();
+
   return (
-    <Flex style={{ justifyContent: "center" }}>
-      <Logo width={200} />
-    </Flex>
+    <Flex 
+      justify="center" 
+      align="center"  
+      style={{ 
+        background: `linear-gradient(to bottom right, #e7f0fe, ${colors.primaryLight} 60%, ${colors.secondary})`,
+        height: '100vh',
+        padding: !sm ? '0 15px' : '0 30px' 
+      }}
+    >
+        <Row style={{ ...loginCardStyle, minWidth: lg ? '900px' : '0',}}>
+          <Col 
+            xs={24} 
+            md={24}
+            lg={12} 
+            style={{ ...flexCentered, padding: !sm ? '15px' : '20px' }}
+          >
+                <LogoH width={230}/>
+                
+                <h3 style={{ color: '#8898aa', fontWeight: '500', marginTop: !md ? '1rem' : '2rem' }}>
+                  Boas-Vindas ao <span style={{ color: colors.primary, textTransform: 'uppercase', marginLeft: 3 }}>Projeto Raízes</span>
+                </h3>
+          </Col>
+
+          <Col 
+            xs={24} 
+            md={24}
+            lg={12} 
+            style={{ ...flexCentered, backgroundColor: '#e7ecfe', padding: !sm ? '15px' : '20px', borderRadius: lg ? '0 8px 8px 0' : '10px',  }}
+          >
+                <Form
+                  name="basic"
+                  labelCol={{ span: 8}}
+                  wrapperCol={{ span: 24 }}
+                  style={{ maxWidth: 600, width: !md ? '100%' : '75%' }}
+                  initialValues={{ remember: true }}
+                  onFinish={onFinish}
+                  autoComplete="off"
+                >
+                  <h2 style={{ ...loginFormTitleStyle, marginTop: !lg ? '-20px' : '-30px' }} >Log In</h2>
+                  <Form.Item
+                    name="username"
+                    rules={[{
+                        required: true,
+                        message: 'Não pode ser vazio!',
+                      }]}
+                  >
+                    <Input
+                      placeholder="Usuário"
+                      size="large"
+                      style={{ borderColor: loginError ? '#ff4d4f' : '' }}
```

Comment:
Colocar cor dentro do colors (se for a cor padrão para erro, chamar de erro). Não necessariamente toda cor precisa estar no colors, mas é interessante se elas se repetirem.